### ISSUE TITLE
[Suggestion] Add warnings for drives that have been changed

### ISSUE THREAD

**Issue by patyieh** [ocmarin/ssd-bot#15]
Not sure if you can do this or if you'd need access to the Spreadsheet, but I think it'd be helpful to include a warning about SSDs that have had parts changed or known issues.

The examples I know are of the ADATA SX8200, WD SN550, PNY XLR8, and the Crucial MX500's write issue.

**Comment by arcasinky** [comment]
I agree.  It would be very nice to include a caveat when listing the specs of a drive that is known to have multiple parts configurations.
